analyzer/quality-analyzer/src/main/java/eu/fasten/analyzer/qualityanalyzer/MetadataUtils.java

```diff
@@ -0,0 +1,273 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package eu.fasten.analyzer.qualityanalyzer;
+
+import eu.fasten.analyzer.qualityanalyzer.data.*;
+
+import eu.fasten.core.data.metadatadb.MetadataDao;
+
+import eu.fasten.core.data.metadatadb.codegen.tables.*;
+import eu.fasten.core.data.metadatadb.codegen.tables.records.FilesRecord;
+import eu.fasten.core.data.metadatadb.codegen.tables.records.PackageVersionsRecord;
+import eu.fasten.core.data.metadatadb.codegen.tables.records.PackagesRecord;
+
+import org.jooq.DSLContext;
+import org.jooq.Record;
+import org.jooq.Result;
+
+import org.json.JSONObject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+
+public class MetadataUtils {
+
+    private final Logger logger = LoggerFactory.getLogger(MetadataUtils.class.getName());
+
+    private Map<String, DSLContext> dslContexts = null;
+
+    private DSLContext selectedContext = null;
+
+    public MetadataUtils(Map<String, DSLContext> contexts) {
+        this.dslContexts = contexts;
+    }
+
+    public void freeResource() {
+        for(String key: dslContexts.keySet()) {
+            dslContexts.get(key).close();
+        }
+    }
+
+    /**
+     *
+     * @param forge         String which could have value MVN, PyPI or C.
+     * @param jsonRecord    Object that contains quality analysis metadata.
+     */
+    public void insertMetadataIntoDB(String forge, JSONObject jsonRecord) {
+
+        if(forge.equalsIgnoreCase(QAConstants.MVN_FORGE)) {
```

Comment:
We already have a constants class which you can use. It's in the `core/data/Constants` and also have `mvnForge`.

Reply:
Correct, I have already updated QAConstants and synched with Constants. However, I wait for the final decision about the forges, before updating it altogether. BTW, Constants do not have any reference to either C or Python forge.

Reply:
OK, done.